vignettes/statistical-specification.Rmd

```diff
@@ -354,6 +354,106 @@ of the distribution is used. This is because this has the same interpretation as
 non-centred parameterisation and setting the "random effects" term to be 0.
 
 
+
+
+## Claret-Bruno Model
+
+
+$$
+\begin{align*}
+y_{ij} &\sim \mathcal{N}(SLD_{ij},\ SLD_{ij}^2 \sigma^2) \\ \\
+SLD_{ij} &= 
+\begin{cases} b_i e^{g_i t_{ij}} & 
+\text{if } t < 0, \\ 
+b_i  \cdot \exp\left(g_i  t_{ij} - \frac{p_i}{c_i}  \left(1 - e^{-c_i t_{ij} }\right)\right) & 
+\text{if } t \geq 0. \end{cases}\\
+\\
+b_i &\sim \text{LogNormal}(\mu_{bl(i)}, \omega_b) \\
+g_i &\sim \text{LogNormal}(\mu_{gk(i)}, \omega_g) \\
+c_i &\sim \text{LogNormal}(\mu_{ck(i)}, \omega_c) \\
+p_i &\sim \text{LogNormal}(\mu_{pk(i)}, \omega_p) \\
+\end{align*}
+$$
+
+Where:
+
+* $i$ is the subject index
+* $j$ is the visit index
+* $y_{ij}$ is the observed tumour measurements
+* $SLD_{ij}$ is the expected sum of longest diameter for subject $i$ at time point $j$
+* $t_{ij}$ is the time since first treatment for subject $i$ at visit $j$
+* $b_i$ is the subject baseline SLD value.
+* $g_i$ is the subject tumour growth rate.
+* $c_i$ is the subject treatment resistance rate.
+* $p_i$ is the subject treatment growth inhibition response.
+* $k(i)$ is the treatment arm index for subject $i$
+* $l(i)$ is the study index for subject $i$
+* $\mu_{\theta k(i)}$ is the population mean for parameter $\theta$ in group $k(i)$
+* $\omega_{\theta}$ is the population variance for parameter $\theta$.
+
+
+### Derivative of the SLD Trajectory Link 
```

Comment:
I double checked this formula and it is correct, one idea is that for simplicity you could reuse the `SLD(t_{ij})` expression in the formulas.